ui/js/filters.js

```diff
@@ -144,3 +147,61 @@ treeherder.filter('alertStatus', [
 treeherder.filter('encodeURIComponent', function() {
     return window.encodeURIComponent;
 });
+
+/*
+ * Given a bug id, fetch the status and summary from
+ * bugzilla for use in tooltips on links to that bug.
+ */
+function initMouseOver(bugLink) {
+    var bugsCacheEntry = bugsCache[bugLink.getAttribute("data-bugid")];
+    if(bugsCacheEntry) {
+        setTitleAndClearListeners(bugLink, bugsCacheEntry);
+    } else {
+        hoverTimeout = window.setTimeout(getBugInfo, 1000, bugLink);
+    }
+}
+
+function cancelMouseOver() {
+    window.clearTimeout(hoverTimeout);
+}
+
+function getBugInfo(bugLink) {
+    var bugID = bugLink.getAttribute("data-bugid");
+    var bugURL = 'https://bugzilla.mozilla.org/rest/bug/' + bugID + '?include_fields=status,summary';
+
+    if(bugID) {
+        // Fetch the bug information from bugzilla and store it locally
+        $.getJSON(bugURL)
```

Comment:
Instead of using a function like this, we should create a ``Bugzilla`` model to call these endpoints.  It may well come in handy in other parts of our code-base, too.